tests/e2e/default/transitions/create-user-for-contacts.replace-user.wdio-spec.js

```diff
@@ -386,7 +386,8 @@ describe('Create user for contacts', () => {
         expect(cookie.value).to.include(newUserSettings.name);
       });
 
-      it('creates new user from latest replace_user form data if multiple are submitted before syncing', async () => {
+      // eslint-disable-next-line max-len
+      it.skip('creates new user from latest replace_user form data if multiple are submitted before syncing', async () => {
```

Comment:
Break the line, but don't skip the lint. 
```suggestion
      it.skip('creates new user from latest replace_user form data'
        + 'if multiple are submitted before syncing', async () => {
```

Or

```suggestion
      it.skip('creates new user from latest replace_user form, if multiple are submitted before sync', async () => {
```